tests/test_dataframe.py

```diff
@@ -1,23 +1,43 @@
 """this module contains tests for the DataFrame class"""
 
+import uuid
+
 import numpy as np
+import pandas as pd
 import pytest
 from deeporigin.data_hub import api
 from deeporigin.data_hub.dataframe import DataFrame
 
-from tests.utils import clean_up_test_objects, minimal_config
+from tests.utils import clean_up_test_objects
+
+TEST_PREFIX = "tc-"
 
-TEST_PREFIX = "tc-x1zrncgf6F-"
```

Comment:
these unique IDs are now dynamically generated, so that tests can be run in parallel